travispy/entities/hook.py

```diff
@@ -30,3 +31,29 @@ class Hook(Entity):
         'private',
         'admin',
     ]
+
+    def _set_hook(self, flag):
+        '''
+
+        '''
+        url = self._session.uri + '/hooks/{}'.format(self.id)
+        data = {
+            "hook": {
+                "active": flag
+            }
+        }
+
+        response = self._session.put(url, json = data)
```

Comment:
You don't need to convert data to ``json`` (I guess), that's why ``data=data`` should do the work.